lasagne/tests/layers/test_special.py

```diff
@@ -633,8 +633,8 @@ def ParametricRectifierLayer(self):
     @pytest.fixture
     def init_alpha(self):
         # initializer for a tensor of unique values
-        return lambda shape: (np.arange(np.prod(shape)).reshape(shape)) \
-            / np.prod(shape)
+        return lambda shape: ((np.arange(np.prod(shape)).reshape(shape)) /
+                              np.prod(shape)).astype(theano.config.floatX)
```

Comment:
Uh! I hadn't noticed when reviewing #446. This is wrong -- it will always return an array of zeros in Python2. We could add a `__future__` import, but I'd prefer to fix this as:
```python
from lasagne.utils import floatX
return lambda shp: (floatX(np.arange(np.prod(shp)).reshape(shp)) /
                      floatX(np.prod(shp)))
```
In theory, I should do a separate PR for this, but then you'd need to undo your change and rebase -- can you just edit it with my fix?